### PR TITLE
style(ui): Update empty states

### LINKED ISSUE #533: Update empty page template
### Global template updates
- [x] Change button to a "primary" (pink) button
- [x] Add a secondary link option under the button instead of a second button
- [x] Change illustration to be a 48x48 SVG placeholder.

---

### Empty Profile
image
- [x] Remove "Register so your friends can find you! [Register handle]" from empty profile state
- [x] Change text to "There’s nothing here yet, get started by starting your first project."
- [x] Change button text to "Start your first project"
- [x] Add "Or register your handle" as the secondary link
- [x] illustration should be [this potted plant illustration](https://www.dropbox.com/s/o343a2dfvsrtoqr/empty-profile.svg?dl=0) which is not a standard emoji, it's a mashup I made specifically for this context.

---

### Other empty states
- [ ] [Revision empty state](https://www.figma.com/file/dnYlebm0ehgyJyT3Po9Kcl/Upstream?node-id=9726%3A545)
- [x] [Issues empty state](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8F%97-Components?node-id=1983%3A0)
- [ ] [Registered profile empty state](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8F%97-Components?node-id=1983%3A0)
- [ ] [Untracked profile empty state](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA/%F0%9F%8F%97-Components?node-id=1983%3A0)

### PR DESCRIPTION
**Pull request by juliendonck**
Addresses parts of #533.

### CODE REVIEW

File: ui/DesignSystem/Component/EmptyState.svelte [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<script>
+  import Icon from "./EmptyState/Icon.svelte";
+  import { Button, Text } from "../Primitive";
+
+  export let icon = "tent";
+  export let text = "Nothing to see here";
+  export let mainCtaText = "Go back";
```
Comment (rudolfs) [inline ui/DesignSystem/Component/EmptyState.svelte:7]: My preference is always to spell things out, I find it reduces confusion when reading code.

Could we maybe call the props `primaryActionText` and `secondaryActionText`?

File: ui/DesignSystem/Component/EmptyState.svelte [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  export let mainDataCy = null;
```
Comment (rudolfs) [inline ui/DesignSystem/Component/EmptyState.svelte:8]: I think we don't need props for the dataCy values as they will always point to the same elements, we just need to adjust the tests accordingly.

File: ui/DesignSystem/Component/EmptyState.svelte [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  export let mainCtaAction = null;
```
Comment (rudolfs) [inline ui/DesignSystem/Component/EmptyState.svelte:9]: The idiomatic Svelte way of doing callbacks is to define custom events for components: https://svelte.dev/examples#component-events.

Here's an example in our own codebase on how to do it:
https://github.com/radicle-dev/radicle-upstream/blob/master/ui/DesignSystem/Component/SegmentedControl.svelte

File: ui/DesignSystem/Component/EmptyState.svelte [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<script>
+  import Icon from "./EmptyState/Icon.svelte";
```
[the remark this answers is not in the document]
Comment (juliendonck) [reply ui/DesignSystem/Component/EmptyState.svelte]: addressed in [c75dea1](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/c75dea1177170aaf6a834c49f591ab8bc80d8988)

File: ui/DesignSystem/Component/EmptyState.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/EmptyState.svelte]: addressed in (ed0bd95)[https://github.com/radicle-dev/radicle-upstream/pull/537/commits/ed0bd952a76832c19aa7cdd6c8d6f783d0cc78d3]

File: ui/DesignSystem/Component/EmptyState.svelte (hunk 3 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/EmptyState.svelte]: addressed in [ed0bd95](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/ed0bd952a76832c19aa7cdd6c8d6f783d0cc78d3)

File: ui/DesignSystem/Component/EmptyState.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/EmptyState.svelte]: addressed in [21e8a4b](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/21e8a4bc9af3b03944c82ed1b50173ee238188a2)

File: ui/Screen/Profile/Projects.svelte [hunk 5]
```diff
@@ -94,8 +96,20 @@
...
+      primaryActionText="Start your first project"
+      secondaryActionText="Or register your handle"
+      on:primaryAction={create}
+      on:secondaryAction={register} />
   {:else}
-    <Onboard />
+    <EmptyState
+      text="There’s nothing here yet, get started by starting your first
+      project."
+      primaryActionText="Or register your handle"
```
Comment (rudolfs) [inline ui/Screen/Profile/Projects.svelte:111]: This should be "Create a new project".

Comment (juliendonck) [reply ui/Screen/Profile/Projects.svelte]: addressed in [7ad5dc2](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/7ad5dc27b7afa4429abe1f5ed60cd4531358614a)

**Pull request merged**